client/src/components/Form/FormDisplay.vue

```diff
@@ -165,6 +165,7 @@ export default {
             });
         },
         onChangeForm() {
+            this.formInputs = JSON.parse(JSON.stringify(this.formInputs));
```

Comment:
Two things regarding this:
* use `structuredClone` instead of the JSON parse/stringify workaround to make a deep copy of an object
* I recently removed this, because it is a hacky way to force a reactive update from vue, which isn't always reliable. If you need something to update and it dosen't, that is because some data changed in a non-reactive way. Tracking this place down and replacing the data mutation with a `set` function triggers an update from vue without the need for a hack to force an update.